packages/x-date-pickers/src/internals/models/muiPickersAdapter.d.ts

```diff
@@ -1,7 +1,7 @@
 import { IUtils } from '@date-io/core/IUtils';
 
 // TODO: Maybe we should add the same constraint.
-// @ts-expect-error TDate in our codebase does not have the `ExtendableDateType` constraint.
+// @ts-ignore TDate in our codebase does not have the `ExtendableDateType` constraint.
```

Comment:
I would have preferred `@ts-expect-error` but it throwed an error on my test application...